src/content/docs/build/tokens/token-customization.mdx

```diff
@@ -40,15 +40,15 @@ Apart from your own [custom properties](/properties/work-with-properties/propert
 ### Add claims to access tokens
 
 You can:
-- add an organization name to an access token
-- add roles to an access token
-- add an email to an access token
-- add an external organization ID to an access token
+- add an organization name to an access token (`org_name`)
+- add roles to an access token (`roles`)
+- add an email to an access token (`email)
```

Comment:
Missing a ` at the end of 'email'

Reply:
![image](https://github.com/user-attachments/assets/a3cff83a-068f-4dff-8f2f-30d3b7d3c3e0)